doc/apimanagerbackup-reference.md

```diff
@@ -0,0 +1,111 @@
+# APIManagerBackup reference
+
+The following Custom Resources are provided:
+
+`APIManagerBackup`
+
+This resource is the resource used to backup a 3scale API Management solution
+deployed using an APIManager custom resource.
+
+## Backup scenarios scope
```

Comment:
Where does it belong?

Reply:
Up to you. `operator-backup-and-restore.md` or somewhere else?

Reply:
If you look at the documentation, from operator-backup-and-restore.md I link to this reference scopes.
Do you want me to move those to operator-backup-and-restore.md? what about the other information like what is backed up/restored that it is also in the reference documentation?

It somewhat seemed it might be too verbose to be in the general operator-backup-and-restore.md documentation and I decided to link to the reference of those sections from the general documentation

Reply:
Ok, up to you